test/e2e/extended/cat-b-scenarios.feature

```diff
@@ -0,0 +1,198 @@
+Feature: Extended category B test scenarios
+
+   Scenario: Examiner completes a passed test with driver faults
+      Given I am logged in as "mobexaminer1" and I have a test for "Miss Florence Pearson"
+      When I start the test for "Miss Florence Pearson"
+      And the candidate enters a new email address
+      And the candidate confirms their communication preference
+      Then I should see the "Declaration - Florence Pearson" page
+      And the candidate completes the declaration page
+      And I proceed to the car
+      Then I should see the "Florence Pearson" page
+      And I complete the waiting room to car page
+      Then I should see the "Test report - Florence Pearson" page
+      When I add a "Accelerator" driver fault
+      And I add a "Timed" driver fault
+      And I add a "Clearance" driver fault
+      And I add a "Signalling" driver fault
+      And I add a "Timed" driver fault
+      Then the driver fault count is "5"
+      And I complete the test
+      And I continue to debrief
+      Then I should see the Debrief page with outcome "Passed"
+      When I end the debrief
+      Then I should see the "Test debrief - Florence Pearson" page
+      And I complete the pass details
+      And I complete the health declaration
+      Then I am on the back to office page
+      And I continue to the office write up
+      Then I should see the "Office" page
+      And I complete the office write up
+      And I upload the test
+      Then I should see the "Journal" page
```

Comment:
Wondering whether we can do anything here around confirming that the upload is successful, but this will probably require some additional work to clear out potential duplicates. Worth some thought.

Reply:
Yep agreed, adds value moving forwards and will look to implement something in a future change.